### ISSUE TITLE
Tests are broken

### ISSUE THREAD

**Issue by bessarabov** [regru/Net-Domain-ExpireDate#3]
Tests fails:

```
bessarabov@lol:~/Dropbox/git/Net-Domain-ExpireDate$ prove -l
t/01.t .. 1/? # .com .net .org tests
# .ru tests
# creation date tests
# domdates tests
# The following tests requires internet connection and may fail if checked domains were renewed...
t/01.t .. 67/? Can't recognise expiration date format: Domain Information: [\xE3\x83\x89\xE3\x83＜\x82ゃ\x83恰\x83\x85\xE5\xA0\xB1]
[Domain Name]                   NIC.JP

[\xE7\x99脂\x8C画\x80\x85\xE5\x90\x8D]                      筝\x80\xE8\x88㋜ぞ\xE5\x9Bｆ\xB3\x95篋堺\x97ユ\x9C㋘\x83\x8D\xE3\x83\x83\xE3\x83\x88\xE3\x83壔\x83若\x82壔\x82ゃ\x83潟\x83\x95\xE3\x82í\x83＜\x83若\x82激\x83с\x83潟\x82祉\x83潟\x82帥\x83\xBC
[Registrant]                    Japan Network Information Center

[Name Server]
[Signing Key]

[\xE7\x99脂\x8C峨拘\xE6\x9C\x88\xE6\x97\xA5]                    2003/07/31
[\xE6\x9C\x89\xE5\x8A号\x9C\x9F\xE9\x99\x90]                      2015/07/31
[\xE7\x8A倶\x85\x8B]                          Active
[\xE6\x9C\x80腟\x82\xE6\x9B贋\x96\xB0]                      2014/08/01 01:05:11 (JST)

Contact Information: [\xE5\x85㋞\x96\x8B\xE9\x80ｇ機腦\x93\xE5\x8F\xA3]
[\xE5\x90\x8D\xE5\x89\x8D]                          筝\x80\xE8\x88㋜ぞ\xE5\x9Bｆ\xB3\x95篋堺\x97ユ\x9C㋘\x83\x8D\xE3\x83\x83\xE3\x83\x88\xE3\x83壔\x83若\x82壔\x82ゃ\x83潟\x83\x95\xE3\x82í\x83＜\x83若\x82激\x83с\x83潟\x82祉\x83潟\x82帥\x83\xBC
[Name]                          Japan Network Information Center
[Email]                         secretariat@nic.ad.jp
[Web Page]
[\xE9\x83灸梢\xE7\x95ɱ\x8F\xB7]                      101-0047
[篏\x8F\xE6\x89\x80]                          \xE6\x9D延根\xE9\x83遵\x8D\x83篁ｇ\x94医\x8C阪\x86\x85腑\x9E\xE7\x94\xB03-6-2
\xE3\x82≪\x83若\x83\x90\xE3\x83潟\x83\x8D\xE3\x83\x83\xE3\x83\x88腑\x9E\xE7\x94違\x83\x93\xE3\x83\xAB4F
[Postal Address]                Urbannet-Kanda Bldg. 4F
3-6-2 Uchi-Kanda
Chiyoda-ku, Tokyo, Japan
[\xE9\x9B肢º\xE7\x95ɱ\x8F\xB7]                      03-5297-2311
[FAX\xE7\x95ɱ\x8F\xB7]                       03-5297-2312

Can't recognise creation date format

#   Failed test at t/01.t line 111.
#          got: ''
#     expected: '2003-07-31'
t/01.t .. 72/? # Looks like you failed 1 test of 72.
t/01.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/72 subtests

Test Summary Report
-------------------
t/01.t (Wstat: 256 Tests: 72 Failed: 1)
  Failed test:  68
  Non-zero exit status: 1
Files=1, Tests=72,  6 wallclock secs ( 0.06 usr  0.06 sys +  0.74 cusr  0.25 csys =  1.11 CPU)
Result: FAIL
bessarabov@lol:~/Dropbox/git/Net-Domain-ExpireDate$
```

**Comment by nalobin** [comment]
whois.jprs.jp changed encoding to utf8. Will be fixed in Net::Whois::Raw 2.82.

**Comment by bessarabov** [comment]
Thank you!
